controllers/controller.js

```diff
@@ -0,0 +1,12 @@
+const { getTopics } = require('../models/model');
+
+function retreiveTopics(req, res, next) {
+    getTopics()
+    .then((result) => {
+        res.status(200).send({topics: result});
+    }).catch((error) => {
+        console.log(error);
```

Comment:
you shouldn't be logging any errors caught here, they need to be handled by passing them to your error handling middleware